### DISCUSSION TITLE
Is possible to add a user (service account) to an oganization without having to accept an email invite?

### DISCUSSION THREAD

**Discussion by christian-unr** [community/community#148189] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I have an organization that I created and would like to add a service account to be able to connect the Git Integration through the Power BI service. However, it seems like when inviting members to an organization on GitHub, they have to accept the invite through email.

What if the account doesn't have a mailbox to accept this invite? Are there any workarounds for a situation like this?

**Comment by divyamlathiya** [comment, marked as answer]
If the service account doesn’t have a mailbox to accept the invite, unfortunately, GitHub requires email confirmation for invitations. There isn't a direct workaround to bypass the email acceptance step.

However, you can try one of these alternatives:

**1. Use a temporary email address:** Create a temporary or disposable email address just to accept the invitation.

**2. Use a GitHub App:** If you only need access for automation purposes, consider using a GitHub App with the necessary permissions, which doesn't require email confirmation.
